lib/ecto/migration/schema_migration.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Ecto.Migration.SchemaMigration do
-  # Define a schema that works with the a table, which is schema_migrations by default
```

Comment:
Started off just meaning to fix the small typo here (`with the a table`), but ended up reading through the `Ecto.Migration` docs as well and fixing up the first section there.